vnc_auto.html

```diff
@@ -151,13 +151,17 @@
         // Set parameters that can be changed on an active connection
         rfb.viewOnly = readQueryVariable('view_only', false);
         rfb.scaleViewport = readQueryVariable('scale', false);
+        if (myRFB == null) {
+            myRFB = rfb;
+        }
     </script>
 </head>
 
 <body>
     <div id="top_bar">
         <div id="status">Loading</div>
         <div id="sendCtrlAltDelButton">Send CtrlAltDel</div>
+        <div id="sendClipboardButton">Send Clipboard</div>
```

Comment:
Check with design if we can use some image here for clipboard and paste buttons.

Reply:
Images are added now